src/foam/nanos/auth/Relationships.js

```diff
@@ -10,7 +10,7 @@ foam.RELATIONSHIP({
   targetModel: 'foam.nanos.auth.Permission',
   forwardName: 'permissions',
   inverseName: 'groups',
-  junctionDAOKey: 'groupPermissionJunctionDAO'
+  junctionDAOKey: 'localGroupPermissionJunctionDAO'
```

Comment:
I don't think we want to do this. If we do, I think client-side code related to relationships will try to use `localGroupPermissionJunctionDAO` from the client and fail since it isn't served.

Reply:
@TW80000 why would client side code be accessing this dao?

Reply:
If you go to the DetailView for either of those models (Group and Permission) and try to edit the relationship using the GUI, it's going to try to read from and write to the junctionDAO, which won't be served if you change this to the local version. It would also break the permission matrix, which needs to read from groupPermissionJunctionDAO on the client to know which permissions the different groups have.